libraries/transformer/src/androidTest/java/androidx/media3/transformer/mh/HdrEditingTest.java

```diff
@@ -139,6 +140,43 @@ public void export_transmuxHlg10File() throws Exception {
     assertThat(actualColorTransfer).isEqualTo(C.COLOR_TRANSFER_HLG);
   }
 
+  @Test
+  public void export_transmuxDolbyVisionFile() throws Exception {
+    Context context = ApplicationProvider.getApplicationContext();
+
+    if (Util.SDK_INT < 24) {
```

Comment:
OK, I'll change the code.

Reply:
You can also use `@SdkSuppress(minSdkVersion = 24)` if your only condition is SDK level.

https://developer.android.com/reference/androidx/test/filters/SdkSuppress

This is preferable to `assumeTrue` imo because it skips the test "earlier", and just doesn't run it at all, instead of needing to mark it as 'skipped' or 'passed' (which has mixed support in different test running environments).

Reply:
OK, I think this method is better. I'll change the code again.